tests/cases/fourslash/importNameCodeFix_noDestructureNonObjectLiteral.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="fourslash.ts" />
+
+// @target: es2015
+// @strict: true
+// @esModuleInterop: true
+
+// @Filename: /array.ts
```

Comment:
silly question: you don't need the leading `/`, do you? I never use it but see tests that do, and I have no idea what the difference is.

Reply:
without leading `/` the `array.ts`'s full path will be `/tests/cases/compiler` I think.

Reply:
No you don’t, but leaving it off scares me because I don’t understand what assumptions are being made about the file system and it makes me uncomfortable

Reply:
I was just imagining writing *real files* to `/` and that scared me.

Reply:
macOS root is no longer user-writable so the test would have failed for me 😁